OlxProject/pages/user_page.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../tools/all_requires'
+
+class UserPage
+  include TopPart
+  USER_PAGE_MESSAGE = "Объявления"
+
+  protected def observed_search_link
+    $driver.find_element(id: 'observed-search-link')
+  end
+
+  def observed_search_link_click
+    observed_search_link().click
```

Comment:
any need for empty brackets?

Reply:
I didn't understand where exactly empty brackets were needed, now I know and already fixed this, thnx!